.husky/pre-commit

```diff
@@ -1,2 +1,3 @@
 npm test
 npx prettier . --check
+npx eslint --fix .
```

Comment:
Think the fix won't be useful because it will only fix in the working directory not in the index of what's about to be comitted.

```suggestion
npx eslint .
```

Reply:
thanks bud fixed it